microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -1235,11 +1372,106 @@ Get information about running WebView2 processes, exiting processes, and failed
 ---
 
 
+<!-- todo: ~ move 2 to here -->
+<!-- ------------------------------ -->
+#### Frame process info (incoming 2)
+<!-- todo: maybe move to near "iframes" instead of in Process management -->
+
+The Frame Process Info API, including `GetProcessExtendedInfos`, provides a snapshot collection of all frames that are actively running in the associated renderer process.  This API enables your app to detect which part of WebView2 is consuming resources such as memory or CPU usage.
+
+See also:
+<!-- todo -->
```

Comment:
this see also is blank. I think we can remove it?

Reply:
removed